include/pops/kernel_types.hpp

```diff
@@ -74,6 +82,22 @@ inline DispersalKernelType kernel_type_from_string(const std::string& text)
         return DispersalKernelType::Uniform;
     else if (text == "deterministic-neighbor" || text == "deterministic_neighbor")
         return DispersalKernelType::DeterministicNeighbor;
+    else if (text == "power law" || text == "power-law")
```

Comment:
This is great. My only small suggestion and this goes for the previously implemented kernels as well (something I should have thought of earlier) is to make it so that we also account for capitalization (e.g. include both "gamma" and "Gamma").